components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/rs/resource-mappings.ts

```diff
@@ -1,10 +1,11 @@
 import { Resource } from "./resource";
+import { HttpController } from "./resource-http-controller";
 
 
 export class ResourceMappings {
-    resources: any = {};
-    controller: any;
-    execute: any;
+    resources: {} = {};
+    controller: HttpController;
+    execute: Function;
```

Comment:
I guess this changes should be reviewed by @vmutafov, as he is more familiar with the `http/rs` API.